File: Folder Python/elektronik.py
class Elektronik:
    def __init__(self, id, nama, jenis, lokasi, rating, komen):
        self.id = id
        self.nama = nama
        self.jenis = jenis
        self.lokasi = lokasi
        self.rating = rating
        self.komen = komen

    def get_id(self):
        return self.id
    
    def get_nama(self):
        return self.nama
    
    def get_jenis(self):
        return self.jenis
    
    def get_lokasi(self):
        return self.lokasi
    
    def get_rating(self):
        return self.rating
    
    def get_komen(self):
        return self.komen
    
    def set_nama(self, nama):
        self.nama = nama
    
    def set_jenis(self, jenis):
        self.jenis = jenis
    
    def set_lokasi(self, lokasi):
        self.lokasi = lokasi
    
    def set_rating(self, rating):
        self.rating = rating
    
    def set_komen(self, komen):
        self.komen = komen

    def display(self):
        print(f"ID: {self.id} | Nama: {self.nama} | Jenis: {self.jenis} | Lokasi: {self.lokasi} | Rating: {self.rating} | Komen: {self.komen}")

File: Folder PHP/elektronik.php
<?php
class Elektronik {
    private $id;
    private $nama;
    private $jenis;
    private $lokasi;
    private $rating;
    private $komen;
    private $gambar; // path file lokal

    public function __construct($id, $nama, $jenis, $lokasi, $rating, $komen, $gambar = '') {
        $this->id = $id;
        $this->nama = $nama;
        $this->jenis = $jenis;
        $this->lokasi = $lokasi;
        $this->rating = $rating;
        $this->komen = $komen;
        $this->gambar = $gambar;
    }

    // Getter
    public function getId() { return $this->id; }
    public function getNama() { return $this->nama; }
    public function getJenis() { return $this->jenis; }
    public function getLokasi() { return $this->lokasi; }
    public function getRating() { return $this->rating; }
    public function getKomen() { return $this->komen; }
    public function getGambar() { return $this->gambar; }

    // Setter
    public function setNama($nama) { $this->nama = $nama; }
    public function setJenis($jenis) { $this->jenis = $jenis; }
    public function setLokasi($lokasi) { $this->lokasi = $lokasi; }
    public function setRating($rating) { $this->rating = $rating; }
    public function setKomen($komen) { $this->komen = $komen; }
    public function setGambar($gambar) { $this->gambar = $gambar; }
}
?>
